controllers/dspipeline_controller_func_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build test_all || test_functional
+// +build test_all test_functional
+
 /*
```

Comment:
how do you feel about moving `suite_test.go` and `dspipeline_controller_func_test.go` to a new folder `tests/functional` or `functional_tests` 

you would need to update some paths though: [[1]](https://github.com/opendatahub-io/data-science-pipelines-operator/blob/498a0f9832c071e6d5b3c58130db175dcdd9f5a1/controllers/suite_test.go#L105), [[2]](https://github.com/opendatahub-io/data-science-pipelines-operator/blob/498a0f9832c071e6d5b3c58130db175dcdd9f5a1/controllers/suite_test.go#L132), [[3]](https://github.com/opendatahub-io/data-science-pipelines-operator/blob/498a0f9832c071e6d5b3c58130db175dcdd9f5a1/controllers/testutil/util.go#L37)

Reply:
we should make a seperate issue for that

Reply:
Created - https://github.com/opendatahub-io/data-science-pipelines-operator/issues/289